output_template/python/lmnet/utils/output.py

```diff
@@ -159,6 +160,9 @@ def __call__(self, outputs, raw_images, image_files):
             "results": [],
         }
 
+        if self.bench is not None and type(self.bench) is dict:
```

Comment:
I don't think it is necessary to guarantee the type of `self.bench` here. (I'm a little uncomfortable to care only `self.bench` specially.)

If you want to add a check mechanism, type annotation looks better way.

Reply:
Fixed and rebased. could you please check?